bindings/python/iota_sdk/wallet/account.py

```diff
@@ -564,6 +564,12 @@ def claim_outputs(
             self, output_ids_to_claim: List[OutputId]) -> Transaction:
         """Claim outputs.
         """
+        return self.prepare_claim_outputs(outputs_ids_to_claim).send()
+
+    def prepare_claim_outputs(
+            self, output_ids_to_claim: List[OutputId]) -> Transaction:
+        """Claim outputs.
+        """
         return Transaction.from_dict(self._call_account_method(
```

Comment:
```suggestion
    def prepare_claim_outputs(
            self, output_ids_to_claim: List[OutputId]) -> PreparedTransaction:
        """Claim outputs.
        """
       return PreparedTransaction(self, self._call_account_method(
```
not sure if that's working or if the prepared data needs to be defined first

Reply:
Yeah, that was indeed broken. Fixed in bd558d45dfdc1027854b7bc240b713f1dcc7ace4